README.md

```diff
@@ -98,7 +116,8 @@ function initAnswers() {
     apiKey: '<API_KEY_HERE>',
     // Required, the key used for your Answers experience
     experienceKey: '<EXPERIENCE_KEY_HERE>',
-    // Required, initialize components here, invoked when the Answers component library is loaded/ready
+    // Optional, initialize components here, invoked when the Answers component library is loaded/ready.
+    //    If components are not added here, they must be added when the init promise resolves
```

Comment:
definitely just my opinion but "must be added" feels a little too strong for me here. technically, you can also add components in both places and things will work fine. Maybe just "they can also" instead of "they must be"?

Reply:
Sure, that sounds good